lib/Mojolicious/Renderer.pm

```diff
@@ -14,6 +14,7 @@ has [qw(compress default_handler)];
 has default_format => 'html';
 has encoding       => 'UTF-8';
 has [qw(handlers helpers)] => sub { {} };
+has min_compress_size => '860';
```

Comment:
nit: small thing, but why make this a string instead of an int? When comparing in the `>=` below, Perl will then make a number scalar value in addition to the string one. It's only three bytes, so not like it really matters, but why not create it as a number if that's how we intend to use it?